### PR TITLE
refactor: add anchor active style (refs SFKUI-8039)

### PR DESCRIPTION
**Pull request by NicklasWei**
Lägger till `active` style för länk baserat på skiss. Skiss visar ingen animering för interaktionerna, gjorde test med det men tycker inte det blir bra då det är så liten visuell skillnad vid active/hover, så blir inte så snärtig då.

Provade att lägga till tester för hover/active men lyckades inte få dem att fungera riktigt.

Todo:
- [x] Checka av designen med UX
- [x] Krav

### CODE REVIEW

**Comment by atfkassan** [comment]
Testat på: https://forsakringskassan.github.io/designsystem/pr-preview/pr-1489/components/links/anchor.html#
Om man gör tab till länken och trycker enter så får man ingen effekt. (Tillgänglighet)

**Comment by atfkassan** [comment]
Testat på Länk i tabell: https://forsakringskassan.github.io/designsystem/pr-preview/pr-1489/examples/table/#/columntypes
- Hela cellen är klickbar yta men klick-effekten får man bara om man klickar på själva länkytan. 
- Får inte heller effekten när man väljer med enter

**Comment by NicklasWei** [comment]
> Testat på: https://forsakringskassan.github.io/designsystem/pr-preview/pr-1489/components/links/anchor.html# Om man gör tab till länken och trycker enter så får man ingen effekt. (Tillgänglighet)

Min research pekar på att ett `a` element inte aktiverar `:active` vid tangentbordsinteraktion, enbart musclick. Såvidare vi inte vill använda javascript för att åtgärda detta har jag inte hittat nån work-around.

File: packages/design/src/core/mixins/_anchor.scss
Comment (NicklasWei) [inline packages/design/src/core/mixins/_anchor.scss:1]: Varför behövs en mixin för anchor? Verkar ju inte vara nåt som återanvänds i fkui. Kan jag bara ta bort den och lägga stylingen tillsammans med anchor istället?

Comment (ext) [reply packages/design/src/core/mixins/_anchor.scss:1]: Om jag minns rätt så fanns det en tanke i tidiga dagar av FKUI att allt skulle vara mixins så att en konsument själv skulle kunna skapa custom selectorer:

```scss
.awesome-anchor {
  @include fkui.ancor();
}
```

(Och kanske anpassa den efter behov)

Det kan också vara så att det används av internwebben, kanske gör en kodsökning internt och se om den nyttjas där.

Om den inte används så tycker jag vi ska ta bort det.

File: packages/vue/src/design-component-tests/Anchor/examples/AnchorVisualTest.vue
Comment (ext) [inline packages/vue/src/design-component-tests/Anchor/examples/AnchorVisualTest.vue:1]: Kanske lägga till texter med mellanslag istället för bara "Link", och en länk som har en väldigt lång text som radbryts, och kanske tom en brödtext med en text som radbryts.

Båda sistnämna är ju corner-cases vid haft stora problem med tidigare.

File: packages/vue/src/design-component-tests/Anchor/Anchor.cy.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import AnchorVisualTest from "./examples/AnchorVisualTest.vue";
+
+describe("Visual", () => {
+    it("anchor should have correct style", () => {
```
Comment (ext) [inline packages/vue/src/design-component-tests/Anchor/Anchor.cy.ts:4]: Vi använder block-varianten överallt här, det kanske ska framgå tydligare? Om du inte lägger in det i brödtext som jag föreslog i andra kommentaren.

**Review by ext — APPROVED** [review]

File: packages/design/src/core/mixins/_anchor.scss
Comment (NicklasWei) [reply packages/design/src/core/mixins/_anchor.scss:1]: Alright, I see. Jag söker igenom och dubbelkollar.

Comment (NicklasWei) [reply packages/design/src/core/mixins/_anchor.scss:1]: Mixin används så låter den vara.

**Comment by dvitamin** [comment]
Vi borde kanske även testa de som använder mixinen, om det nu är fk.se t ex.

File: packages/design/src/core/mixins/_anchor.scss [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
     color: $anchor-color-text-standard-default;
-    text-decoration: underline;
-    text-decoration-thickness: $anchor-underline-width;
-    text-underline-offset: $anchor-underline-offset;
-    text-decoration-color: $anchor-color-text-standard-default;
     font-weight: var(--f-font-weight-medium);
     line-height: var(--f-line-height-large);
+    text-decoration-color: $anchor-color-text-standard-default;
+    text-decoration-line: underline;
+    text-decoration-skip-ink: none;
```
Comment (NicklasWei) [inline packages/design/src/core/mixins/_anchor.scss:9]: Underline bröts i vissa fall vid `active` style, så stänger av det beteendet. Då vår vanliga offset är under underhängande bokstäver så påverkar det dock ingenting förutom för `active`.
